### ISSUE TITLE
Error while creating a new ML Workspace

### ISSUE THREAD

**Issue by osama222** [hning86/azuremlps#4]
Hey guys, I've followed the steps to install and import the DLL and I'm using a self signed-cert as advised and passing the required parameters to create the new ML Workspace as below
New-AmlWorkspace -AzureSubscriptionId $azureSubscriptionId -ManagementCertThumbprint $mgmtCertThumb -WorkspaceName 'ABCD' -Location $location -storageAccountName $storageAccountName -StorageAccountKey $storageAccountKey -OwnerEmail $ownerEmail

It work working perfectly until last 2 days and then suddenly started throwing below error

----------------------------
New-AmlWorkspace : One or more errors occurred.
At line:1 char:1 + New-AmlWorkspace -AzureSubscriptionId $azureSubscriptionId -Managemen ...+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  + CategoryInfo          : NotSpecified: (:) [New-AmlWorkspace], AggregateException  + FullyQualifiedErrorId : System.AggregateException,AzureML.PowerShell.NewWorkspace
-----------------------------
 
And it's not even telling me where's the error coming from or which parameter is wrong although I'm sure nothing is wrong since I didn't change anything and i'm sure the certificate is still valid, the subscription is valid and i can create a new workspace from the portal.
Can you please help on this.

**Comment by hning86** [comment]
Can you put a Fiddler session in place to capture the error message going over the wire?

**Comment by hning86** [comment]
This is fixed in the beta 0.2.4 release.

**Comment by osama222** [comment]
I've used fiddler with the latest 0.2.4 release and I got 403 error The server failed to authenticate the request. verify that the certificate is valid and is associated with this subscription.
I've double checked the certificate validity and it's valid and associated with this subscription.
Is there's a way to pass the value of the parameter **-ManagementCertThumbprint** via *.publishsettings file or any other way.
Thanks
